src/modules/auto-copy-on-click.js

```diff
@@ -0,0 +1,50 @@
+/**
+ *  @author Deux Huit Huit
+ *
+ */
+(function ($, undefined) {
+	
+	'use strict';
+
+	var site = $('#site');
+	var sels = {
+		toCopy: '.js-text-to-copy',
+		copyBtn: '.js-copy-on-click-btn',
+		message: '.js-confirm-message'
+	};
+	var timer = null;
+
+	var onClick = function () {
+		var delay = 1500;
+		
+		site.find(sels.toCopy).each(function () {
+			var t = $(this);
+			t.select();
+			document.execCommand('copy');
+		});
+
+		App.modules.notify('changeState.update', {
+			item: site.find(sels.message),
```

Comment:
Multiples item may be catched. changeState is only able to manage one item at a time